src/cascadia/TerminalSettingsModel/Theme.idl

```diff
@@ -78,6 +78,7 @@ namespace Microsoft.Terminal.Settings.Model
 
         // A helper for retrieving the RequestedTheme out of the window property
         Windows.UI.Xaml.ElementTheme RequestedTheme { get; };
+        Boolean IsActuallyDarkTheme();

```

Comment:
Ideally, we could consolidate these 2 functions. Any way we can do that?

Reply:
Personally hate the name `IsActuallyDarkTheme` but I don't think there's really a better option haha

Reply:
See, these are two different things. `RequestedTheme` could be `system`, which means we need to do the hacky color lookup to see if we're _actually_ in dark theme or not.

Reply:
It puts too many smarts in the dumb model objects for me FWIW

Reply:
personally, I wouldn't _want_ my model objects reasoning about system state

Reply:
I agree, I'm not sure if the model objects should know too much about system stuff. I'd personally they'd be entirely stateless and "dumb", just like when using MVC patterns.

Reply:
Oh I see it's because `_isSystemInDarkTheme()` actually already exists inside `TerminalSettings`... right. Nvm, ignore me then. 😅